src/plugins/event-plugins/__tests__/DomEventPlugin.test.ts

```diff
@@ -255,4 +255,24 @@ describe('DomEventPlugin tests', () => {
             })
         );
     });
+    test('DomEventPlugin registers new DOM events when plugin is updated', async () => {
```

Comment:
suggestion: Verify the DOM event is recorded instead of verifying the config is updated.

i.e., `when a DOM event is added at runtime then the DOM event is recorded`

Reply:
Refactored and renamed.